Zotero/Scenes/AppCoordinator.swift

```diff
@@ -218,9 +233,17 @@ final class AppCoordinator: NSObject {
 
     private func showRestoredState(for data: RestoredStateData) {
         guard let mainController = self.window?.rootViewController as? MainViewController,
-              let (url, library) = self.loadRestoredStateData(forKey: data.key, libraryId: data.libraryId) else { return }
-        DDLogInfo("AppCoordinator: show restored state - \(data.key); \(data.libraryId); \(url.relativePath)")
-
+              let (url, library, collection) = self.loadRestoredStateData(forKey: data.key, libraryId: data.libraryId, collectionId: data.collectionId) else { return }
+        if let collection {
+            DDLogInfo("AppCoordinator: show restored state - \(data.key); \(data.libraryId); \(data.collectionId); \(url.relativePath)")
+            mainController.showItems(for: collection, in: library, isInitial: false)
+        } else {
+            DDLogInfo("AppCoordinator: show restored state - \(data.key); \(data.libraryId); \(url.relativePath)")
+            // Collection is missing, show all items instead
+            let collection = Collection(custom: .all)
+            mainController.showItems(for: collection, in: library, isInitial: true)
```

Comment:
I'd probably even show a warning/error here, because the collection should be available.

Also last question, why is `isInital: true` here and `isInitial: false` above on 239?

Reply:
`true` doesn't save the selected collection to defaults.

Reply:
Check https://github.com/zotero/zotero-ios/pull/718/commits/5e9013080fe4609379b740eacb93f7b89d94dd40

Reply:
Might be worth renaming `isInitial` to something else then, doesn't make sense when `isInitial` is `true` and `false` at the same time based on whether the collection is available.

Reply:
Check https://github.com/zotero/zotero-ios/pull/718/commits/2a1d6715294a69792e65ad6012d7c6a273dcb875